### PR TITLE
Adds needed permissions for deploy user

### PR DESCRIPTION
**Pull request by dornera**
Deploy user needs additional permissions to update the cluster task

### CODE REVIEW

File: apps/cantaloupe/cantaloupe.tf [hunk 1]
```diff
@@ -80,15 +80,68 @@ module "fargate" {
 ###################
 ### Deploy user ###
 ###################
+data "aws_iam_policy_document" "deploy_policy" {
+  # allows user to push/pull to the registry
+  statement {
```
Comment (gravesm) [inline apps/cantaloupe/cantaloupe.tf]: Why not just use the readwrite policy from the ecr module?

Comment (dornera) [reply apps/cantaloupe/cantaloupe.tf]: It needed more policies for other resources, so I thought it was best to through it all into one policy_document that we might user later for another deploy user. Potentially a tf-mod-iam-deploy user module?

Is it better to take out the ECR part of the doc and then attach that to the user along with his additional doc?

Comment (gravesm) [reply apps/cantaloupe/cantaloupe.tf]: I'd attach it separate rather than duplicating, because then what's the point of the module?

Comment (dornera) [reply apps/cantaloupe/cantaloupe.tf]: 👍

**Pull request merged**